rollup.config.js

```diff
@@ -22,14 +28,14 @@ export default [
 			}
 		],
 		plugins: [
-			replace({
-				VERSION: pkg.version,
-				delimiters: ["<@", "@>"]
-			}),
+			replace(replaceVersionConfig),
 			ts({
-				module: "ES2020"
+				module: "es2020"
 			}),
-			resolve()
+			resolve(),
+			copy({
```

Comment:
Would be a bit more elegant to just inline the content of package-esm.json here rather than have it as a separate file

Reply:
I agree, but didn't see a way to do that.